app/adapters/application.js

```diff
@@ -1,3 +0,0 @@
-import LFAdapter from 'ember-localforage-adapter/adapters/localforage';
```

Comment:
Actually, I think you were trying to rename the file application.js to adapter.js or similar. But you forgot to add the new file to your git commit.